lib/app_server.py

```diff
@@ -86,6 +83,7 @@ def prepare_args(self, args=[]):
 
     def deploy(self, vardir=None, silent=True, need_init=True):
         self.vardir = vardir
+        self.workdir = self.vardir
```

Comment:
It is something unrelated to the PR?

Reply:
Or some renaming?

Reply:
Ok, got it, that is split of vardir (scripts and so on) and workdir of tarantool. I suggest to mention it in the commit message, because it is unobvious refactoring.